model/trace/database.yaml

```diff
@@ -33,6 +30,12 @@ groups:
           conditionally_required: >
             If readily available. Otherwise, if the instrumentation library parses `db.query.text` to capture
             `db.collection.name`, then it SHOULD be the first collection name found in the query.
+        note: >
+          If the collection name is parsed from the query, it SHOULD match the value provided
+          in the query and may be qualified with the schema and database name.
```

Comment:
#nit: Is this note related to this PR?

Reply:
Great catch, it's redundant since it matches the note in the registry